app/controllers/application_controller.rb

```diff
@@ -3,9 +3,8 @@
 class ApplicationController < ActionController::API
   include RescueFromUnlessLocal
 
-  # will delete this as soon as we have anonymous users
-  # we'll always have user ids
-  TEMP_ANONYMOUS = '7c56ebf5-a850-44b1-a5e5-8e3866f6a248'
+  # Sentinel value so we'll always have user and device ids
+  ANONYMOUS_UUID = '00000000-0000-0000-0000-000000000000'
```

Comment:
we're now returning nil right? instead of a sentinel

Reply:
yup that's the plan. Figuring out the  spec now.

Reply:
Can't find a way to make a spec that fails the same way as @TomWoodward  reported. Changed code to return nil.